### ISSUE TITLE
Add WASM to CI

### ISSUE THREAD

**Issue by ryanisaacg** [ryanisaacg/quicksilver#238]

**Comment by memoryruins** [comment]
While adding wasm to travis ci config, I ran into the following issue https://github.com/rust-lang-nursery/rand/issues/503. A PR has already been opened https://github.com/rust-lang-nursery/rand/pull/505. Quite the timing haha.

**Comment by ryanisaacg** [comment]
Yeah, I'd say we have to remove it if we want to add wasm. I would prefer to be able to leave it for the desktop targets but if we can't that's fine.

**Comment by dhardy** [comment]
The issue with Rand is that we choose to remove a stub function which allowed the project to compile, but panicked on usage. If we add that back (https://github.com/rust-lang-nursery/rand/pull/505) then your CI test should *compile* but any attempt to use `thread_rng` / `OsRng` or one of the other ways to get random numbers will just result in a panic. Is that what you want?

**Comment by ryanisaacg** [comment]
@dhardy What would be required for actual support for the Rand crate on wasm?

**Comment by dhardy** [comment]
Yes, including `stdweb` will work, and is probably the solution you want.

**Comment by ryanisaacg** [comment]
Sounds good, as soon as we get stdweb support landed then this should be right after.
